src/netspresso_trainer/dataloaders/classification/local.py

```diff
@@ -8,7 +8,7 @@
 class ClassificationCustomDataset(BaseCustomDataset):
 
     def __init__(self, conf_data, conf_augmentation, model_name, idx_to_class,
-                 split, samples, transform=None, with_label=True):
+                 split, samples, transform=None, with_label=True, **kwargs):
```

Comment:
There are added `**kwargs` on each datasets(classification, detection, segmentation), but there is no addition on `BaseCustomDataset` which is a parent class.
How about adding `**kwargs` in `BaseCustomDataset` also?

Reply:
I think it depends on the preference of code style.

I personally believe that kwargs in BaseClass make hard to understand the content. Rather, it should only used for specific classes.

Especially, there's no initialization with task-specific elements in BaseClass. I think it is not that late if it is needed to add kwargs in BaseClass and inits with it.